OnDemandPassenger/Constants.m

```diff
@@ -6,6 +6,6 @@
 #import "Constants.h"
 
 #warning Replace with your app id
-NSString * APP_ID = @"YOUR_APP_ID";
```

Comment:
Mmm.. I see my comments don't show up nicely in the unified view. My point was to amend the first commit to remove the credentials instead of adding a new commit at the end to delete them.